src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.length) {
+    return {};
+  }
+
+  const stringToObject = sourceString
+    .split(';')
+    .map((value) => value.split(':').map((spaces) => spaces.trim()))
```

Comment:
The chained method calls should have the same indentation to improve readability. Align `.split()`, `.map()`, and `.reduce()` at the same level.